releases/unreleased/delete-task-on-error.yml

```diff
@@ -0,0 +1,11 @@
+---
+title: Remove tasks that fail to be scheduled
```

Comment:
You should add another changelog file explaining we are adding the `add_scheduled_task` to the api